source/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ endif()
 
 option(USE_SDL2 "Build using SDL2" OFF)
 option(GAME_MODULES_ONLY "Only build game modules" OFF)
+option(USE_STEAMWORKS_SDK "Include the Steamworks SDK" OFF)
```

Comment:
Disabled by default, because we can't distribute steamworks sdk.

Reply:
What if we want Steamworks builds from appveyor or travis?

Reply:
then there needs to be some buildstep that downloads a zip or whatever and extracts that in-place.